pyproject.toml

```
@@ -21,7 +21,7 @@ packages = [{ include = "qibotn", from = "src" }]
 
 [tool.poetry.dependencies]
 python = "^3.9,<3.13"
-qibo = { git = "https://github.com/qiboteam/qibo.git" }
+qibo = ">=0.2.8"
```

```suggestion
qibo = "^0.2.8"
```
This would also deny the next major